tests/integration/ovs_test.py

```diff
@@ -103,3 +117,38 @@ def test_create_and_remove_ovs_bridge_with_internal_port_and_static_ip():
 
     assertlib.assert_absent(BRIDGE1)
     assertlib.assert_absent(PORT1)
+
+
+@pytest.mark.xfail(raises=NmstateLibnmError, strict=True)
+def test_nm_ovs_plugin_missing():
+    with disable_nm_ovs_plugin():
+        libnmstate.apply(
+            {
+                Interface.KEY: [
+                    {
+                        Interface.NAME: BRIDGE1,
+                        Interface.TYPE: InterfaceType.OVS_BRIDGE,
+                        Interface.STATE: InterfaceState.UP,
+                    }
+                ]
+            }
+        )
+
+
+@contextmanager
+def disable_nm_ovs_plugin():
+    libcmd.exec_cmd(DNF_REMOVE_NM_OVS_CMD)
+    libcmd.exec_cmd(SYSTEMCTL_RESTART_NM_CMD)
+    ret = [1,0,0]
+    while ret[0]:
+        ret = libcmd.exec_cmd(SYSTEMCTL_IS_ACTIVE_NM_CMD)
+        time.sleep(1)
```

Comment:
I took a deeper look. It seems that NM is actually working already but the dbus code in Nmstate needs to be re-initialized.

This code makes the test work:
`libnmstate.nm.checkpoint._nmdbus_manager = None`

I guess the same is needed after the second restart. Maybe @EdDev has a good idea how to solve this properly. 

Ideally (IMHO) we would create a separate container without the OVS plugin and run the test in there but this would mean a bigger effort.

On the other hand, I guess this discovered a bug that the Nmstate code will fail when a long-running process loaded libnmstate and NM is restarted in-between. So it might be worth fixing this. @EdDev is vdsm a long-running process that loaded libnmstate? What will it do when NetworkManager is restarted while it is running?

It might be worth adding a test with a simple state and restarting NM to show that this breaks NM (however, it will also break all following tests, so we would need to container separation again...).

What are your opinions?